Corona-Warn-App/src/androidTest/java/testhelpers/BaseUITest.kt

```diff
@@ -15,6 +15,9 @@ abstract class BaseUITest : BaseTestInstrumentation() {
         Manifest.permission.WRITE_EXTERNAL_STORAGE
     )
 
+    @get:Rule
+    val systemUIDemoModeRule = SystemUIDemoModeRule()
+
```

Comment:
Did you perform a successful screenshot CI run after the UI test adjustments?

Reply:
I ran them locally ,but here a CI [Build](https://app.circleci.com/pipelines/github/corona-warn-app/cwa-app-android/15601/workflows/ed40c2d7-52d3-4533-a905-cfda09534041/jobs/125127)